OMCompiler/Compiler/Template/CodegenC.tpl

```diff
@@ -2857,8 +2857,8 @@ template generateStaticSparseData(String indexName, String systemType, list<tupl
   case _ then
       let sp_size_index =  lengthListElements(unzipSecond(sparsepattern))
       let sizeleadindex = listLength(sparsepattern)
-      let colPtr = genSPCRSPtr(listLength(sparsepattern), sparsepattern, "colPtrIndex")
-      let rowIndex = genSPCRSRows(lengthListElements(unzipSecond(sparsepattern)), sparsepattern, "rowIndex")
+      let colPtr = genSPCRSPtr(sizeleadindex, sparsepattern, "colPtrIndex")
```

Comment:
Error: Elaborated expression 'l_sizeleadindex' of type 'Tpl.Text' failed to type adapt to its inferred type 'Integer'.